content/guides/trash/index.md

```diff
@@ -0,0 +1,17 @@
+---
+rank: 50
+related_endpoints: []
+related_guides: []
+required_guides: []
+related_resources:
+  - folders
+  - files
+  - web-links
+---
+
+# Trash
+
+Box uses a two-stage process to remove or "trash" files, folders, or web links
```

Comment:
Would it be worth adding a little intro to just define what the trash is? Just a simple one-liner and then a break to this? e.g. "Before items are deleted they might end up in a user's trash. The trash can be found in the UI but also via the API..." (please rewrite to your liking).

Also please feel free to refer to file/folder/web link as item(s). We do it all over the docs.